backend/src/express.ts

```diff
@@ -0,0 +1,27 @@
+import express, {Request, Response } from 'express';
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+import morgan from 'morgan';
+import cors from 'cors';
+import apiRouter from './routes/api.router';
+// import healthRoute from './routes/health';
+
+const app = express();
+
+// Express middleware
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+// app.use(cookieParser());
+// app.use(compression());
+// app.use(morgan('dev'));
+app.use(cors());
+
+app.get('/api', (req: Request, res: Response) => {
+  res.send('Hello, TypeScript Express root');
+});
```

Comment:
If the Quickstart template doesn't require this, I'd recommend just removing it. 
If the Quickstart template DOES require this, it's doing the job of your `health` endpoint, making that no longer required!